natlas-server/app/errors/handlers.py

```diff
@@ -1,24 +1,50 @@
-from flask import render_template
+import json
+
+from flask import render_template, request, Response
 from app.errors import bp
 from app import db
 import elasticsearch
 import sentry_sdk
 
 
+def json_response(status: int, msg: str) -> str:
+    err = json.dumps({"status": status, "message": msg})
```

Comment:
Hey sir, not sure if you noticed, but this is technically part of your API contract that you've added here, but what's the forcing function to ensure that you use a consistent format. Can we share an object, method, something to make sure this "API contract schema" is actually enforced and shared?

Reply:
Technically this is enforced and shared because all of these handlers return build_response, and build_response always calls it the same way. So the keys in the dict will always be the same so long as we exit through an error handler.

Reply:
The always call build_response? https://github.com/natlas/natlas/blob/2d99d2ead1f970fb381e992a6eef2de94414de99/natlas-server/app/api/routes.py#L28
Or this? https://github.com/natlas/natlas/blob/2d99d2ead1f970fb381e992a6eef2de94414de99/natlas-server/app/api/routes.py#L48
This? https://github.com/natlas/natlas/blob/2d99d2ead1f970fb381e992a6eef2de94414de99/natlas-server/app/api/routes.py#L68
This? https://github.com/natlas/natlas/blob/2d99d2ead1f970fb381e992a6eef2de94414de99/natlas-server/app/api/routes.py#L111
https://github.com/natlas/natlas/blob/2d99d2ead1f970fb381e992a6eef2de94414de99/natlas-server/app/api/routes.py#L141
https://github.com/natlas/natlas/blob/2d99d2ead1f970fb381e992a6eef2de94414de99/natlas-server/app/api/routes.py#L192